ktor-client/ktor-client-tests/posix/src/io/ktor/client/tests/utils/ClientLoader.kt

```diff
@@ -16,17 +16,22 @@ actual abstract class ClientLoader {
      */
     actual fun clientTests(
         skipPlatforms: List<String>,
+        skipEngines: List<String>,
```

Comment:
Nice! Let's complete this idea: `skipPlatforms` is obsolete. Let's remove it.

I would use an engine name instead of `fqname` here.

Reply:
Good point, done.